zipkin-lens/pom.xml

```diff
@@ -34,11 +34,11 @@
     <!-- Update occasionally based on LTS, but don't overrun what's in Alpine:
          https://nodejs.org/en/download/
          https://pkgs.alpinelinux.org/packages?name=nodejs&branch=edge -->
-    <node.version>14.15.3</node.version>
-    <exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
-    <frontend-maven-plugin.version>1.11.0</frontend-maven-plugin.version>
-    <maven-clean-plugin.version>3.1.0</maven-clean-plugin.version>
-    <maven-resources-plugin.version>3.2.0</maven-resources-plugin.version>
+    <node.version>14.21.3</node.version>
```

Comment:
anything beyond this needs a new lock file and also fails for reasons not entirely unexpected ;)